src/CBT.NuGet/Tasks/GenerateNugetProperties.cs

```diff
@@ -115,6 +120,7 @@ public bool Execute(string packageRestoreFile, string[] inputs, string propsFile
             }
             catch (Exception e)
             {
+                Log.LogError(e.ToString());
                 Trace.TraceError(e.ToString());
```

Comment:
The issue I have with the silent behavior is that you can have a failure in your build at some place and not notice or see the symptoms much further downstream and it will be much more difficult to trace back to where the real problem is.